gradlew

```diff
@@ -205,6 +205,12 @@ set -- \
         org.gradle.wrapper.GradleWrapperMain \
         "$@"
 
+# Stop when "xargs" is not available.
+if ! command -v xargs >/dev/null 2>&1
+then
+    die "xargs is not available"
+fi
+
```

Comment:
QUES: are these all auto-generated with the new version of gradle?

Reply:
Yes, all of this is generated by running `./gradlew wrapper --gradle-version 7.5.1`. I made no manual code changes in this PR.